src/StoreKeeper/WooCommerce/B2C/Backoffice/Notices/AdminNotices.php

```diff
@@ -140,7 +141,7 @@ private function StoreKeeperOrderCheck()
     {
         global $post_type;
 
-        if ($this->isPostPage() && 'edit' === $this->getRequestAction() && 'shop_order' === $post_type) {
+        if ($this->isPostPage() && 'edit' === $this->getRequestAction() && in_array(['shop_order', DataSynchronizer::PLACEHOLDER_ORDER_POST_TYPE], $post_type, true)) {
```

Comment:
is it wise to use the internal classes ? What i they just remove it out of blue :P ?